Assets/Scripts/UI/GyroCheck.cs

```diff
@@ -0,0 +1,36 @@
+using UnityEngine;
+
+public class GyroCheck : MonoBehaviour
+{
+    private GlobalManager globalManager;
+    private AudioManager audioManager;
+    public GameObject GyroCheckMenu;
+
+    private void Awake()
+    {
+        // Finds our audio manager
+        audioManager = FindObjectOfType<AudioManager>();
+        // Find our global manager
+        globalManager = GameObject.FindWithTag("GlobalManager").GetComponent<GlobalManager>();
```

Comment:
In BlockBlasterz we don't have global manager, nor we'll reintroduce it, for Block racers it's fine , but in Blasters create something like an OptionsManager that should be a singleton